app/src/main/java/com/codepath/flixster/MovieActivity.java

```diff
@@ -23,37 +24,61 @@
   MovieArrayAdapter movieAdapter;
   ListView lvItems;
 
+  private SwipeRefreshLayout swipeContainer;
+
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_movie);
+    swipeContainer = (SwipeRefreshLayout) findViewById(R.id.swipeContainer);
+
+    final AsyncHttpClient client = new AsyncHttpClient();
+
+    // Set up refresh listener
+    swipeContainer.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+      @Override
+      public void onRefresh() {
+        fetchMoviesAsync(client);
+      }
+    });
+
+    // configure the refreshing colors
+    swipeContainer.setColorSchemeResources(android.R.color.holo_blue_bright,
+            android.R.color.holo_green_light,
+            android.R.color.holo_orange_light,
+            android.R.color.holo_red_light);
 
     lvItems = (ListView) findViewById(R.id.lvMovies);
     movies = new ArrayList<>();
```

Comment:
@nesquena Is this array list necessary any more since I'm directly adding items to the adapter in line 70?